.gitignore

```diff
@@ -68,3 +68,24 @@ embedded/
 Gemfile.lock
 .ropeproject/
 setuptools-*.zip
+
+
+### Operating Systems ###
+
+## OSX ##
+*.DS_Store
+.AppleDouble
+.LSOverride
+# Thumbnails
+._*
+
+## Linux ##
+*~
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+# KDE directory preferences
+.directory
```

Comment:
if you open the folder in a kde browser you can get that there. It's not the most likely thing, the nfs thing is also not super likely, but I'm fine with this being a bit overly broad.